sut/drush/sites/synctest.site.yml

```diff
@@ -0,0 +1,14 @@
+remote:
```

Comment:
I suspect that this file, and the similar one above, were committed in error. The current sut does not have these. I think they are generated by the tests and cleaned up at the end.